### DISCUSSION TITLE
Expose the ability to download a backup to be more obvious

### DISCUSSION THREAD

**Discussion by jeffjsmith33** [home-assistant/frontend#18330] [Dashboard]
I was wondering how to get backups from the Home Assistant to my Mac - and only found solutions that seemed more complicated than necessary for the occasional copy.  The, by chance, I double clicked one of my back files from Dashboard/System/Backups and wondered what the 3 dots allowed.  And there, obscurely hidden away, was an option to simply download the backup file from the Home Assistant to my Mac!  

I think it would be helpful to have this option made more visible - especially for novice users.  I suggest it should be shown on the list of backup files screen when a backup file is selected.  It is so very useful!

**Comment by matthiasdebaat** [comment]
I agree. We should move the download and delete backup to the table. 
Created this issue: https://github.com/home-assistant/frontend/issues/18342

**Comment by renarena** [comment]
Especially expose the information about on the fly decryption during download.
This is explained in the docs on common tasks and on the integration backup, but who knows backups are an integration???
There is a help icon on the backup integration page but nothing on the system/backups page.

Proposals how this info could be brought to the user:
• Add Backup information on a more visible place in the docs
• Add a Help link to the backup dashboard
• change the Text "Download Backup" in dot menu to something containing  decrypt
• Add a short sentence above the backup table stating the decryption during downloads.

There are people out there desperately searching for a way to decrypt the backups on there NAS/cloud service to extract a file without restoring the entire backup. 
They find several tools made to decrypt the backups (may work better or not so good), but nothing appears in searches stating the ability to download the backup to get it decrypted.
